### PR TITLE
WIP - fix tools version

### PR DESCRIPTION
**Pull request by marandaneto**
## :loudspeaker: Type of change

- [X] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description
installing build tools 29.x

## :bulb: Motivation and Context
because we are installing build tools 28.x but using 29.x

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] All tests passing

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ before_install:
 android:
   components:
     - platform-tools
-    - build-tools-28.0.3
+    - build-tools-29.0.2
```
Comment (bruno-garcia) [inline .travis.yml]: We tried this before, but not available on Travis yet :\

Comment (tonyo) [reply .travis.yml]: looks like it is available 🤷‍♂ 
image

Comment (marandaneto) [reply .travis.yml]: @bruno-garcia yeah I told @tonyo about that, but we wanted to try it again as it shows in the log

Comment (bruno-garcia) [reply .travis.yml]: I see. So it was finally added?!
Builds are hanging though

Comment (marandaneto) [reply .travis.yml]: build passed and build logs also show them being downloaded

**Pull request merged**